2.0/UnlitTest/glTF/UnlitTest.gltf

```diff
@@ -0,0 +1,155 @@
+        }
+    ],
+    "asset" : {
+        "copyright": "Copyright 2019 Analytical Graphics, Inc, CC-BY 4.0 https://creativecommons.org/licenses/by/4.0/ - Model by Ed Mackey.",
+        "generator" : "Khronos Blender glTF 2.0 I/O, with hand-edits",
+        "version" : "2.0"
+    },
+    "bufferViews" : [
+        {
+            "buffer" : 0,
+            "byteLength" : 1152,
+            "byteOffset" : 0
+        },
+        {
+            "buffer" : 0,
+            "byteLength" : 1152,
+            "byteOffset" : 1152
+        },
+        {
+            "buffer" : 0,
+            "byteLength" : 264,
+            "byteOffset" : 2304
+        }
+    ],
+    "buffers" : [
+        {
+            "byteLength" : 2568,
+            "uri" : "UnlitTest.bin"
+        }
+    ],
+    "extensions" : {},
+    "materials" : [
+        {
+            "name" : "Orange",
+            "pbrMetallicRoughness" : {
+                "baseColorFactor": [
+                    1,
+                    0.217637640824031,
+                    0,
+                    1
+                ]
+            },
+            "extensions": {
+                "KHR_materials_unlit": {}
+            }
+        },
+        {
+            "name" : "Blue",
+            "pbrMetallicRoughness" : {
+                "baseColorFactor": [
+                    0,
+                    0.217637640824031,
+                    1,
+                    1
+                ]
+            },
+            "extensions": {
+                "KHR_materials_unlit": {}
+            }
+        }
+    ],
+    "meshes" : [
+        {
+            "name" : "Orange Mesh",
+            "primitives" : [
+                {
+                    "attributes" : {
+                        "NORMAL" : 1,
+                        "POSITION" : 0
+                    },
+                    "indices" : 2,
+                    "material" : 0
+                }
+            ]
+        },
+        {
+            "name" : "Blue Mesh",
+            "primitives" : [
+                {
+                    "attributes" : {
+                        "NORMAL" : 1,
+                        "POSITION" : 0
+                    },
+                    "indices" : 2,
+                    "material" : 1
+                }
+            ]
+        }
+    ],
+    "nodes" : [
+        {
+            "mesh" : 0,
+            "name" : "Orange Object",
+            "translation": [
+                -1.2,
+                0,
+                0
+            ]
+        },
+        {
+            "mesh" : 1,
+            "name" : "Blue Object",
+            "translation": [
+                1.2,
+                0,
+                0
+            ]
+        }
+    ],
+    "scene" : 0,
+    "scenes" : [
+        {
+            "name" : "Scene",
+            "nodes" : [
+                0,
+                1
+            ]
+        }
+    ],
+    "extensionsUsed": [
+        "KHR_materials_unlit"
+    ],
+    "extensionsRequired": [
```

Comment:
This is true for the specGloss sample also. Is adding extensionsRequired a good thing for the sample models? I didn't do this for TextureTransformTest. At the least, I think the README should mention that the assets have the extensionsRequired set and that if the implementation doesn't support this extension, it is expected to fail to load. Thoughts?

Reply:
@emackey Was this added by glTF-Blender-IO? I could see the argument for requiring the extension in a test file for that extension, but I definitely didn't meant to make it the default from the Blender exporter..

Reply:
Yeah, the case was clear in the specGloss test, where I intentionally removed the metal/rough fallback from the spec/gloss side of the test, to prevent comparisons of metal/rough against itself.  So that model requires the specGloss model to load correctly.

This one is more subjective:  Certainly something will load and display if unlit is not required, but, the clear intention of this model is to display the unlit material itself, which can't be done without the support of the extension.

I'll add a note to the README.

Reply:
README updated.